benchmark/es/spread-bench.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const common = require('../common.js');
+const assert = require('assert');
+
+const bench = common.createBenchmark(main, {
+  method: ['apply', 'spread'],
+  count: [5, 10, 20],
```

Comment:
We can add a config for `context: ['this', 'null']` so we can benchmark against things like `fn.apply(context, args)`, `fn.call(context, ..args)` and `fn2 = fn.bind(context); [...] fn2.call(..args) [...]` too.